### PR TITLE
Update instructions for running locally

### PR DESCRIPTION
**Pull request by lucperkins**
Addresses #112

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-serve:
+serve: get-milestones
```
Comment (austinlparker) [inline Makefile]: IMO we don't want to run this target before serve, because it'll require everyone to have a GitHub API token that can pull repo status.

Comment (MikeGoldsmith) [reply Makefile]: You only need the `npm install` from the _get-milestones_ target. Separating that from the `node -r esm ...` should be sufficient and won't requite GH tokens be configured.

Comment (lucperkins) [reply Makefile]: Done

**Comment by austinlparker** [comment]
thanks, lgtm!

**Pull request merged**